easybuild/easyconfigs/p/Pyomo/Pyomo-5.5.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'Bundle'
```

Comment:
Please use `PythonBundle` easyblock instead of `Bundle`

Reply:
@vanzod Done